packages/mobile/src/walletConnect/saga.ts

```diff
@@ -58,8 +61,56 @@ const TAG = 'WalletConnect/saga'
 
 let client: WalletConnectClient | null = null
 
+function getDefaultSessionTrackedProperties(
+  session: SessionTypes.Proposal | SessionTypes.Created | SessionTypes.Settled
+) {
+  const peer = 'proposer' in session ? session.proposer : session.peer
+  const { name: dappName, url: dappUrl, description: dappDescription, icons } = peer.metadata
+  const {
+    blockchain: { chains: permissionsBlockchains },
+    jsonrpc: { methods: permissionsJsonrpcMethods },
+    notifications: { types: permissionsNotificationsTypes },
+  } = session.permissions
+  const { protocol: relayProtocol } = session.relay
+  return {
+    dappName,
+    dappUrl,
+    dappDescription,
+    dappIcon: icons[0],
+    permissionsBlockchains,
+    permissionsJsonrpcMethods,
+    permissionsNotificationsTypes,
+    relayProtocol,
+  }
+}
+
+function getDefaultRequestTrackedProperties(request: SessionTypes.RequestEvent) {
```

Comment:
There's a `params` argument in the request that gets sent, does it make sense to sent it as well? 

It might not, just checking

Reply:
Yes I was hesitant about this because of privacy concerns.
But there's no PII data in this. Maybe except for the `personal_sign` method, which it seems we should redact.